packages/components/src/components/FunctionChart/index.tsx

```diff
@@ -51,15 +51,16 @@ export const FunctionChart: FC<FunctionChartProps> = ({
   environment,
   height,
 }) => {
-  const parameters = fn.parameters();
-  if (parameters.length !== 1) {
+  const parameters = fn.paramCounts();
+  if (includes(parameters, 1)) {
```

Comment:
Same suggestion about `parameters.includes(...)` as in other comments

Reply:
This should be negated, alert if counts _don't_ include 1.

Reply:
Good catch!